hls/lib/hls_endpoint.ex

```diff
@@ -289,6 +295,19 @@ defmodule Membrane.RTC.Engine.Endpoint.HLS do
         track.origin in endpoints and track.id not in subscribed_tracks
       end)
 
+    origins = MapSet.new(tracks, fn track -> track.origin end)
+
+    not_found_endpoints =
+      Enum.filter(endpoints, fn endpoint ->
+        MapSet.member?(origins, endpoint)
+      end)
+
+    if not_found_endpoints != [] do
+      Membrane.Logger.warning(
+        "Couldn't subscribe on any track from endpoints: #{not_found_endpoints}"
+      )
+    end
+
```

Comment:
It's perfectly fine to subscribe on endpoints before they add tracks. I think that this log is valuable for both debugging and as a server log but I'm not sure if it should be a warning.